lib/containers/common.pm

```diff
@@ -295,6 +295,9 @@ sub check_containers_connectivity {
     record_info "connectivity", "Checking that containers can connect to the host, to each other and outside of the host";
     my $container_name = 'sut_container';
 
+    record_info 'Disable ipv6', 'https://sd.suse.com/servicedesk/customer/portal/1/SD-135489';
+    assert_script_run 'sysctl -w net.ipv6.conf.all.disable_ipv6=1';
```

Comment:
```suggestion
    if (script_run('ping -6 -c 2 google.com') != 0) {
        record_info('Disable ipv6', 'https://sd.suse.com/servicedesk/customer/portal/1/SD-135489', result => 'softfail');
        assert_script_run 'sysctl -w net.ipv6.conf.all.disable_ipv6=1';
    }
```

Let's hope this will get solved one fine day and adjust the test code accordingly

Reply:
:man_facepalming:

Reply:
done